api/Services/DBService.cs

```diff
@@ -270,6 +287,20 @@ public async Task<List<Staff>> GetAllStaff()
 
             return queryDefinition;
         }
+        private QueryDefinition CreateNewGameQueryDefinition(
+            string containerName,
+            Game newEntry)
+        {
+            var queryString = $"INSERT INTO {containerName} (id, title, romanizedTitle, releases," +
+                $" sharedMechanics, series, averagePlayLength, dataPoints, boxArtURL) " +
+                $"VALUES ('{newEntry.Id}', '{newEntry.Title}', '{newEntry.RomanizedTitle}', " +
```

Comment:
We also have this in the other queries, but we'll eventually also want to parameterize these queries instead of putting the passed in directly into the sql strings. This is ripe for sql injection as is. I'd say no action needed at the moment but will want to clean this up when we clean up the other stuff.

Reply:
Sounds good, parameterizing the str for injection would be beyond me though atm.